src/webapp/hooks/useSurveys.ts

```diff
@@ -1,27 +1,48 @@
 import { useEffect, useState } from "react";
-import { Id } from "../../domain/entities/Ref";
 import { Survey, SURVEY_FORM_TYPES } from "../../domain/entities/Survey";
 import { useAppContext } from "../contexts/app-context";
+import { useCurrentSurveys } from "../contexts/current-surveys-context";
 
-export function useSurveys(surveyType: SURVEY_FORM_TYPES, parentSurveyId: Id | undefined) {
+export function useSurveys(surveyFormType: SURVEY_FORM_TYPES) {
     const { compositionRoot } = useAppContext();
     const [surveys, setSurveys] = useState<Survey[]>();
     const [loading, setLoading] = useState(false);
     const [error, setError] = useState<string>();
+    const {
+        currentPPSSurveyForm,
+        currentCountryQuestionnaire,
+        currentHospitalForm,
+        currentWardRegister,
+    } = useCurrentSurveys();
 
     useEffect(() => {
         setLoading(true);
-        compositionRoot.surveys.getSurveys.execute(surveyType, parentSurveyId).run(
-            surveys => {
-                setSurveys(surveys);
-                setLoading(false);
-            },
-            err => {
-                setError(err.message);
-                setLoading(false);
-            }
-        );
-    }, [compositionRoot.surveys.getSurveys, surveyType, parentSurveyId]);
+        let orgUnitId = "";
```

Comment:
Try to avoid using `let` here

Reply:
done in https://github.com/EyeSeeTea/amr-surveys/pull/11